integreat_cms/cms/utils/tree_mutex.py

```diff
@@ -87,10 +93,12 @@ def innermost_function(*args: P.args, **kwargs: P.kwargs) -> R:
             monkey patch :meth:`treebeard.models.Node._get_database_cursor` to get djangos db cursor
             and finally call the decorated ``func``.
             """
-            with lock:
-                with transaction.atomic(using=DEFAULT_DB_ALIAS, durable=False):
-                    with monkeypatch_cursor_func(using=DEFAULT_DB_ALIAS):
-                        return func(*args, **kwargs)
+            with (
+                lock,
+                transaction.atomic(using=DEFAULT_DB_ALIAS, durable=False),
+                monkeypatch_cursor_func(using=DEFAULT_DB_ALIAS),
+            ):
+                return func(*args, **kwargs)
```

Comment:
@PeterNerlich I think this should be fine (ruff would complain, but not auto-fix), but I just wanted to doule check with you

Reply:
yeah looks good